.github/workflows/command-line.yaml

```diff
@@ -0,0 +1,55 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Command Line Hector
+
+# Controls when the action will run. 
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  # This workflow contains two jobs, one that builds hector on ubuntu and the second 
+  # builds and tests Hector on macos. The purpose of this work for is to make sure that Hector 
+  # can be built from the command line. Persumably if it fails a test on one platform it will 
```

Comment:
I'd remove the "Presumably..." sentence. It's not necessarily true, which is why we test on multiple platforms.